src/components/PopupWithForm.js

```diff
@@ -19,8 +20,13 @@ export default class PopupWithForm extends Popup {
     return inputObj;
   }
 
+  _SavingTextOnForm() {
```

Comment:
Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)

Reply:
Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.